docs/api/src/cli/completion.rs.html

```diff
@@ -190,12 +190,12 @@
     }
 
     <span class="kw">let </span>completion_script = <span class="string">r#"
-#compdef cargo make cargo-make
+#compdef _cargo_make makers
```

Comment:
I don't understand how to apply this for sub commands such as `cargo make` and `cargo-make make`.